packages/components/src/common/Notifications/NotificationIcon/NotificationIcon.tsx

```diff
@@ -1,38 +1,44 @@
 import * as React from "react";
 
-import { BannerColor } from "../../../Banner";
-import CheckCircleIcon from "../../../Icons/CheckCircle";
+import CheckCircleRoundedIcon from "../../../Icons/CheckCircleRounded";
 import DangerousRoundedIcon from "../../../Icons/custom/DangerousRounded";
-import ForumIcon from "../../../Icons/Forum";
-import NotificationsIcon from "../../../Icons/Notifications";
-import WarningIcon from "../../../Icons/Warning";
+import ForumRoundedIcon from "../../../Icons/ForumRounded";
+import NotificationsRoundedIcon from "../../../Icons/NotificationsRounded";
+import WarningRoundedIcon from "../../../Icons/WarningRounded";
 import styles from "./NotificationIcon.module.css";
 
+export type NotificationVariant =
+  | "brand"
+  | "neutral"
+  | "success"
+  | "warning"
+  | "alert";
```

Comment:
I could also see this in a types file in `common/Notifications`. Maybe we should even just have one color types file for all the different components. It might be useful to be able to see at a glance which colors the different components are using. Doesn't need to be in this PR of course; just thinking in text.